rust/Cargo.toml

```diff
@@ -3,6 +3,12 @@ name = "xain-fl"
 version = "0.7.0"
 authors = ["XAIN AG <engineering@xain.io>"]
 edition = "2018"
+description = "The XAIN project is building a privacy layer for machine learning so that AI projects can meet compliance such as GDPR and CCPA. The approach relies on Federated Learning as enabling technology that allows production AI applications to be fully privacy compliant."
+license = "Apache-2.0"
+license-file = "../LICENSE"
+keywords = ["federated-learning", "fl"
```

Comment:
"ai", and "machine-learning" as well perhaps?

Reply:
Good point.